src/main/java/nextstep/subway/section/domain/Section.java

```diff
@@ -0,0 +1,74 @@
+package nextstep.subway.section.domain;
+
+import nextstep.subway.common.BaseEntity;
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.station.domain.Station;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+public class Section extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
현재 컨벤션을 봤을때 필드 사이 줄바꿈이 빠진 것 같네요~ :)
```suggestion

    @ManyToOne(fetch = FetchType.LAZY)
```